specs/eip4844/p2p-interface.md

```diff
@@ -247,6 +249,8 @@ disconnect and/or temporarily ban such an un-synced or semi-synced client.
 Clients MUST respond with at least the first blobs sidecar that exists in the range, if they have it,
 and no more than `MAX_REQUEST_BLOBS_SIDECARS` sidecars.
 
+Clients MUST not respond with empty blobs sidecars.
```

Comment:
To clarify what an "empty" blob sidecar is. It's a `BlobSidecar` that does not contain any blobs, but it may contain a non-zero `beacon_block_root` and `beacon_block_slot` per [get_blobs_sidecar](https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/validator.md#sidecar).